extra/man/alacritty.5.scd

```diff
@@ -52,16 +67,21 @@ Windows: _"powershell"_
 
 	Default: _true_
 
+Default:
+
+	live_config_reload = _true_++
+working_directory = _"None"_
```

Comment:
This is just duplicated? We shouldn't have things documented twice.

You can just document the defaults as toml.

Reply:
Hm, I think I just forgot to remove `default`